core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review-modal.controller.spec.ts

```diff
@@ -233,13 +241,10 @@ describe('Question Suggestion Review Modal Controller', function() {
     beforeEach(angular.mock.inject(function($injector, $controller) {
       const $rootScope = $injector.get('$rootScope');
       QuestionObjectFactory = $injector.get('QuestionObjectFactory');
-      SuggestionModalService = $injector.get('SuggestionModalService');
 
       $uibModalInstance = jasmine.createSpyObj(
         '$uibModalInstance', ['close', 'dismiss']);
 
-      spyOnAllFunctions(SuggestionModalService);
```

Comment:
API of spyOnAllFunctions has changed, so I decided to take the opportunity be more explicit rather than continue using it.